cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -330,22 +293,18 @@ function update-objects() {
 # Global variables set by function match-objects.
 nsnames_for_delete=""   # a list of object nsnames to be deleted
 for_update=""           # a list of pairs <nsname>;<filePath> for objects that should be updated
-nsnames_for_ignore=""   # a list of object nsnames that will be ignored
 new_files=""            # a list of file paths that weren't matched by any existing objects (these objects must be created now)
 
 
 # $1 path to files with objects
 # $2 object type in the API (ReplicationController or Service)
-# $3 name separator (single character or empty)
 function match-objects() {
     local -r addon_dir=$1
     local -r obj_type=$2
-    local -r separator=$3
 
     # output variables (globals)
     nsnames_for_delete=""
     for_update=""
```

Comment:
How about `for_update`. I think we don't use it either. Function `update-objects` can also be removed if I understand your intentions correctly.